onchain/src/art_peace.cairo

```diff
@@ -275,7 +286,7 @@ pub mod ArtPeace {
                 self.emit(PixelPlaced { placed_by: caller, pos, day, color });
```

Comment:
This line was a placeholder till ExtraPixelsPlaced was implemented and should be removed.